File: 351004/Bunevich/labs/task330/src/main/java/com/example/demo/dto/response/StoryResponseTo.java
package com.example.demo.dto.response;

import com.fasterxml.jackson.annotation.JsonRootName;
import com.fasterxml.jackson.databind.PropertyNamingStrategies;
import com.fasterxml.jackson.databind.annotation.JsonNaming;
import lombok.Data;

import java.time.LocalDateTime;
import java.util.List;

@Data
@JsonRootName("story")
@JsonNaming(PropertyNamingStrategies.LowerCamelCaseStrategy.class)
public class StoryResponseTo {
    private long id;
    private long writerId;
    private String title;
    private String content;
    private LocalDateTime created;
    private LocalDateTime modified;

    private List<TagResponseTo> tags;
}

File: 351003/Soika/entities-app/publisher/src/main/kotlin/com/example/entitiesapp/dto/StoryResponseTo.kt
package com.example.entitiesapp.dto

import java.io.Serializable

data class StoryResponseTo(
    val id: Long = 0,
    val title: String = "",
    val content: String = "",
    val writerId: Long = 0,
    val markIds: Set<Long> = emptySet()
) : Serializable